_docs/overview/what-is-cloudgov.md

```diff
@@ -24,7 +24,7 @@ cloud.gov runs on top of industry-provided infrastructure ([currently Amazon Web
 
 ## cloud.gov provides Compliance as a Service
 
-cloud.gov was designed with **FISMA compliance** in mind. cloud.gov has a [FedRAMP Joint Authorization Board (JAB) Provisional Authority to Operate (P-ATO)]({{ site.baseurl }}{% link _docs/overview/fedramp-tracker.md %}) at the Moderate impact level. 
+cloud.gov was designed with **FISMA compliance** in mind. cloud.gov has a [FedRAMP Joint Authorization Board (JAB) Provisional Authority to Operate (P-ATO)]({{ site.baseurl }}{% link _docs/overview/fedramp-tracker.md %}) at the Moderate impact level. In most cases, the majority of Low and Moderate controls are managed for you. And because our documentation and controls are always up to date, you can incorporate them directly into any compliance documentation your systems need.
```

Comment:
One more slight phrasing tweak, though it could probably be refined further:

```suggestion
cloud.gov was designed with **FISMA compliance** in mind. cloud.gov has a [FedRAMP Joint Authorization Board (JAB) Provisional Authority to Operate (P-ATO)]({{ site.baseurl }}{% link _docs/overview/fedramp-tracker.md %}) at the Moderate impact level. In most cases, the majority of Low and Moderate controls are managed for you. We also continuously update our documentation and controls with the latest information available so that you can incorporate them directly into any compliance documentation your systems need.
```